saltyrtc/server/bin.py

```diff
@@ -246,3 +246,7 @@ def main():
     finally:
         if obj['logging_handler'] is not None:
             obj['logging_handler'].pop_application()
+
+
+if __name__ == '__main__':
```

Comment:
This works, but only if the line is there:

    python -m saltyrtc.server.bin

I'm aware of the entry point script, but sometimes you want to call the module directly :)